testing/cli/test_sct_deepseg.py

```diff
@@ -38,24 +37,67 @@ def test_model_dict():
         assert 'default' in value
 
 
-@pytest.mark.parametrize('fname_image, fname_seg_manual, fname_out, task', [
+@pytest.mark.parametrize('fname_image, fname_seg_manual, fname_out, task, thr', [
     (sct_test_path('t2s', 't2s.nii.gz'),
      sct_test_path('t2s', 't2s_seg-deepseg.nii.gz'),
      't2s_seg_deepseg.nii.gz',
-     'seg_sc_t2star'),
+     'seg_sc_t2star',
+     0.9),
+    (sct_test_path('t2', 't2.nii.gz'),
+     sct_test_path('t2', 't2_seg-manual.nii.gz'),
+     't2_seg_deepseg.nii.gz',
+     'seg_sc_contrast_agnostic',
+     0.5),
+    (sct_test_path('t2', 't2.nii.gz'),
+     None,  # TODO: implement integrity test (for now, just checking if output segmentation file exists)
+     't2_seg_deepseg.nii.gz',
+     'seg_spinal_rootlets_t2w',
```

Comment:
I'm currently missing a good ground truth file for the rootlets test. I'm not sure... is that something we want to add to `sct_testing_data`?

Reply:
One thing to note is that, visually scanning the output on my placeholder image I used (`sct_testing_data/t2/t2.nii.gz`), it does look like the model identified the rootlets within the image fairly well. So, perhaps we could just store the current output in `sct_testing_data`, as described by @mguaypaq in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4345#discussion_r1457981736?

Reply:
I've gone with this approach for the rootlets ground truth. :)